projects/uriparser/uri_parse_fuzzer.cc

```diff
@@ -101,7 +102,9 @@ extern "C" int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size) {
   UriParserA parser2;
   UriParserStateA state2;
   state2.uri = parser2.get_mutable_uri();
-  if (URI_SUCCESS != uriParseUriA(&state2, uri2.c_str())) return 0;
+  if (URI_SUCCESS != uriParseUriA(&state2, uri2.c_str())) {
```

Comment:
As you'll see in the style resource I sent you separately, we prefer to avoid ["Yoda comparisons"](https://en.wikipedia.org/wiki/Yoda_conditions) like this.  You should swap the order of these around.